test/graphql/mutation/business-additional-details.test.js

```diff
@@ -0,0 +1,335 @@
+config.set('auth.disabled', false)
+import nock from 'nock'
+import { config } from '../../../app/config.js'
+import { transformBusinesDetailsToOrgAdditionalDetailsUpdate } from '../../../app/transformers/rural-payments/business.js'
+import { makeTestQuery } from '../makeTestQuery.js'
+
+const v1 = nock(config.get('kits.gatewayUrl'))
+
+const orgAdditionalDetailsUpdatePayload = {
+  id: 'organisationId',
+  companiesHouseRegistrationNumber: '01234613020',
+  charityCommissionRegistrationNumber: '1111',
+  businessType: {
+    id: 101443,
+    type: 'Not Specified'
+  },
+  dateStartedFarming: '2025-01-01',
+  legalStatus: {
+    id: 102106,
+    type: 'Limited Partnership (LP)'
+  }
+}
+
+const setupNock = () => {
+  nock.disableNetConnect()
+
+  v1.post('/organisation/search', {
+    searchFieldType: 'SBI',
+    primarySearchPhrase: 'sbi',
+    offset: 0,
+    limit: 1
+  }).reply(200, {
+    _data: [
+      {
+        id: 'organisationId'
+      }
+    ]
+  })
```

Comment:
Consider extracting common code for reuse:
```suggestion

const nockOrgSearch = () =>  v1.post('/organisation/search', {
    searchFieldType: 'SBI',
    primarySearchPhrase: 'sbi',
    offset: 0,
    limit: 1
  }).reply(200, { _data: [{ id: 'organisationId' }] })

const setupNock = () => {
  nock.disableNetConnect()
  nockOrgSearch()
```
...then you can replace the 4 other instances of this same code with a call to the `nockOrgSearch` function.

Reply:
done